.github/dependabot.yml

```diff
@@ -5,21 +5,31 @@ updates:
   schedule:
     interval: weekly
   open-pull-requests-limit: 99
+  cooldown:
+    default-days: 7
 - package-ecosystem: "docker-compose"
   directory: "/"
   schedule:
     interval: weekly
   open-pull-requests-limit: 99
+  cooldown:
+    default-days: 7
 - package-ecosystem: "github-actions"
   directory: "/"
   schedule:
     interval: weekly
   open-pull-requests-limit: 99
+  cooldown:
+    default-days: 7
 - package-ecosystem: "npm"
   directory: "/"
   schedule:
     interval: daily
   open-pull-requests-limit: 99
+  cooldown:
+    default-days: 7
+    exclude:
+      - addons-linter
```

Comment:
if we're excluding packages we own, and addons-frontend?  (+ autograph?)

Reply:
Good idea. https://github.com/mozilla/addons-server/pull/23917/commits/1c1c1f5ee7db4ac84580b2f6ad5c0bd5034ae0c6

Reply:
`mozilla-services/`, but 🤷 you could be right

Reply:
What matters is the package/image name. And actually I was wrong, `autograph` image on docker hub is under `mozilla/` like `addons-frontend` is, so it would get the exclusion as well.